.vscode/settings.json

```diff
@@ -20,42 +19,21 @@
     "**/.git": true,
     ".vscode": true,
     ".bundle": true,
-    ".gitignore": false,
-    ".gitpod.yml": false,
-    ".rspec": false,
-    ".devcontainer/": false,
-    "config.ru": false,
-    "Dockerfile": false,
-    "Gemfile": false,
-    "Gemfile.lock": false,
-    "tmp/": false,
-    "install-packages": false,
-    "main.rb": false,
-    "README.md": false,
-    "app.rb": false,
-    "environment.rb": false,
-    "bin/": false,
-    "spec/": false,
-    "examples.txt": false,
-    "config/": false
   },
   "files.insertFinalNewline": true,
   "files.trimFinalNewlines": true,
   "screencastMode.onlyKeyboardShortcuts": true,
   "screencastMode.verticalOffset": 10,
   "explorer.compactFolders": false,
   "explorer.incrementalNaming": "smart",
-  "html.format.endWithNewline": true,
   "html.format.preserveNewLines": true,
   "html.format.templating": true,
   "terminal.integrated.altClickMovesCursor": true,
   "terminal.integrated.defaultProfile.linux": "bash",
   "vscode-erb-beautify.keepBlankLines": 1,
-  "ruby.intellisense": false,
```

Comment:
This appeared to be vestigial. I couldn't find a reason for it.

Reply:
I think this was related to the [`vscode-ruby` extension](https://github.com/rubyide/vscode-ruby) which was deprecated last year. IIRC it was installed by default in Gitpod and maybe Codespaces